### PR TITLE
Add message for incorrect lesson time

### PR DESCRIPTION
**Pull request by H-horizon**
Invalid Commands
Command: add lesson lecture
Output: Invalid format. Please enter a valid day and time!

Command: add lesson lecture ;; Wednesday
Output: Invalid format. Please enter a valid day and time!

Command: add lesson lecture ;; Wednesday 9 am - 10am ;; sadglnasdjkghadfsgfadshb
Output: Invalid link entered.

Valid commands
Command: add lesson lecture ;; Wednesday 9 am - 10am ;;
Output: Added lecture to lesson list.

Command: add lesson lecture ;; Wednesday 9 am - 10am
Output: Added lecture to lesson list.

Command: add lesson lecture ;; Wednesday 9 am - 10am ;; https://nus-sg.zoom.us/j/abc ;;
Output: Added lecture to lesson list.

Command: add lesson lecture ;; Wednesday 9 am - 10am ;; https://nus-sg.zoom.us/j/abc
Output: Added lecture to lesson list.

Command: add lesson lecture ;; Wednesday 9 am - 10am ;; https://nus-sg.zoom.us/j/abc ;; Ivan Chong
Output: Added lecture to lesson list.

Command: add lesson lecture ;; Wednesday 9 am - 10am ;; https://nus-sg.zoom.us/j/abc ;; Ivan Chong ;; 
Output: Added lecture to lesson list.

Command: add lesson lecture ;; Wednesday 9 am - 10am ;; https://nus-sg.zoom.us/j/abc ;; Ivan Chong ;;  ivan_chong@megagmail.com
Output: Added lecture to lesson list.

Ivan, please verify the parser executes the given commands as expected. 
#91

### CODE REVIEW

File: src/main/java/seedu/duke/common/Messages.java [hunk 1]
```diff
@@ -24,18 +24,20 @@
...
-    
+
     public static final String MESSAGE_TASK_FIELDS_EMPTY = "Missing task details.";
     public static final String MESSAGE_INVALID_TASK_DEADLINE = "Invalid/missing deadline.";
-    
+
     public static final String MESSAGE_NON_INTEGER_INDICES = "Warning, non-integer values removed: %s";
     public static final String MESSAGE_OUT_OF_BOUNDS_INDICES = "Warning, out of bounds index removed: %s";
 
+    public static final String MESSAGE_INVALID_DAY_TIME = "Invalid format. Please enter a valid day and time!";
```
Comment (8kdesign) [inline src/main/java/seedu/duke/common/Messages.java:39]: I notice that this is the only file modified push request. Is this message in use?

Comment (H-horizon) [reply src/main/java/seedu/duke/common/Messages.java:39]: Ivan told me to add this to the messages class. He's gonna use it in the parser class

**Comment by ivanchongzhien** [comment]
Looks good, thanks!

**Review by ivanchongzhien — APPROVED** [review]
Looks good, will add it into parser.

**Pull request merged**